powerdns/tests/test_auto_ptr.py

```diff
@@ -61,3 +61,22 @@ def test_alt_ptr_created(self):
         )
         domain = Domain.objects.get(name='1.168.192.in-addr.arpa')
         self.assertTrue(domain.get_soa().content.endswith('1200'))
+
+    def test_ptr_autoremove(self):
+        """A PTR record is automatically removed with its A record"""
+        a = RecordFactory(
+            domain=self.domain,
+            type='A',
+            name='site.example.com',
+            content='192.168.1.1',
+        )
+        Record.objects.get(name='1.1.168.192.in-addr.arpa', type='PTR')
```

Comment:
Maybe: ``self.assertTrue(Record.objects.filter(name='1.1.168.192.in-addr.arpa', type='PTR').exists())``